packages/dashmate/src/commands/config/set.js

```diff
@@ -20,7 +20,11 @@ class ConfigSetCommand extends ConfigBaseCommand {
       optionValue = null;
     }
 
-    config.set(optionPath, optionValue);
+    if (typeof config.get(optionPath) === 'object') {
```

Comment:
It's not so easy with JS. Many types will return type objects but they are not. You better you json schema and type defined there. On other hand, I guess it shouldn't be a problem to use `JSON.parse` for all types, isn't? It would be nice to have tests so we can prove that we didn't break anything.